### ISSUE TITLE
Question: Why are both RLS and rust-analyzer being used together?

### ISSUE THREAD

**Issue by qdrs** [ujihisa/shadow.vim#3]
I have noticed that rust-tools requires rust-analyzer to work properly, however most rust guides for neovim mention that you only need to use rust-tools (which comes with analyzer), and that you shouldn't use RLS. For example here : https://sharksforarms.dev/posts/neovim-rust/ .

However if I disable RLS in shadow.vim the autocomplete doesn't work correctly, it doesn't show me the available methods on an object for example.

So in summary I have 2 questions:
1) Why are you using both together, is it cause of the same issue I am encountering if I don't use RLS?
2) Have you attempted to do it with just rust-analyzer, if so what did you encounter?

Thanks

**Comment by qdrs** [comment]
Sorry wrong repo!
